_posts/en/newsletters/2022-09-28-newsletter.md

```diff
@@ -0,0 +1,150 @@
+  would need to pay proportionally more than a payment which leaves 75%
+  of its outbound capacity available.
+
+    Developer ZmnSCPxj [described][zmnscpxj ratecards] a simple way to
+    use ratecards, "you can model a rate card as four separate channels
+    between the same two nodes, with different costs each.  If the path
+    at the lowest cost fails, you just try at another route that may
+    have more hops but lower effective cost, or else try the same
+    channel at a higher cost."
+
+    The proposal also allows for negative fees.  For example, a channel
+    could subsidize payments forwarded through it when it had more than
+    75% outbound capacity by adding 1 satoshi to every 1,000 satoshi in
+    payment value.  Negative fees could be used by merchants to
+    incentivize others to restore inbound capacity to channels
+    frequently used for receiving payments.
+
+    Neigut notes that some nodes are already adjusting their fees based
+    on channel capacity, and that fee ratecards provides a more
+    efficient way to communicate their intentions to the network than
+    frequently advertising new feerates via the LN gossip network.
+
+- **Bitcoin implementation designed for testing soft forks on signet:**
+  Anthony Towns [posted][towns bi] to the Bitcoin-Dev mailing list a
+  description of a fork of Bitcoin Core he's working on that will only
+  operate on the default [signet][topic signet] and which will enforce
+  rules for soft fork proposals with high quality specification and
+  implementations.  This should make it simpler for users to experiment
+  with the proposed change, including directly comparing one change to
+  another similar change (e.g. comparing [OP_CHECKTEMPLATEVERIFY][topic
+  op_checktemplateverify] to [SIGHASH_ANYPREVOUT][topic
+  sighash_anyprevout]).  Towns also plans to include proposed major
+  changes to transaction relay policy (such as [package relay][topic
+  package relay]).
+
+    Towns is seeking constructive criticism of the idea for this new
+    testing implementation, called [Bitcoin Inquisition][], as well as
+    reviews of the [pull requests][bi prs] adding the initial set of
+    soft forks to it.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Is it possible to fingerprint HD wallets?]({{bse}}115311)
+  Pieter Wuille points out that while identifying UTXOs created using an [HD
+  wallet][topic bip32] is not possible, other onchain data can be used to
```

Comment:
```suggestion
  wallet][topic bip32] is not possible, other on-chain data can be used to
```

Reply:
We previously settled on "onchain", see https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#compound-words